ansible/configs/selinux-policy/post_software.yml

```
@@ -14,7 +14,7 @@
         - "Here are your credentials to access the lab environment:"
         - ""
         - "Password: {{ student_password | d(hostvars[groups.bastions.0].student_password)}}"
-        - "IP Address: {{ hostvars[groups.bastions.0].public_ip_address }}"
+        - "SSH Connection: {{ student_user }}@{{ hostvars[groups.bastions.0].public_ip_address }}"
         - ""
         - "To access your lab environment please follow the guide:"
```

And here change to something like:

```suggestion
        - "To access the lab instructions, check the following guide:"
```